Library/Homebrew/style.rb

```diff
@@ -167,7 +167,16 @@ def run_rubocop(files, output_type,
     def run_shellcheck(files, output_type)
       files = shell_scripts if files.blank?
 
-      args = ["--shell=bash", "--enable=all", "--external-sources", "--source-path=#{HOMEBREW_LIBRARY}", "--", *files]
+      args = [
+        "--shell=bash",
+        "--enable=all",
+        "--external-sources",
+        "--source-path=#{HOMEBREW_LIBRARY}",
+        # TODO: fix these
+        "--exclude=SC2310,SC2311,SC2312",
```

Comment:
@XuehaiPan Would you be able to take a look at these? Thanks ❤️

Reply:
I think we can disable these globally? We probably do not need these checks:

https://github.com/koalaman/shellcheck/wiki/SC2310
https://github.com/koalaman/shellcheck/wiki/SC2311
https://github.com/koalaman/shellcheck/wiki/SC2312

Otherwise, we should add `# shellcheck disable=SCxxxx` (and explanation) to each corresponding line or at the top of the files, as we usually do.

Reply:
@XuehaiPan Yeh, I'm fine with either option. If we don't need these: could you submit a PR to update the comment explaining why?

Reply:
Thinking about it: I think the comment might be better because it'll stop e.g. the VSCode integration complaining, right?

Reply:
Could use `.shellcheckrc`.

Reply:
Yeh, that'd work for me 👍🏻